packages/connect-examples/expo-playground/app/components/common/UnifiedLogger.tsx

```
@@ -0,0 +1,392 @@
+          icon: <AlertCircle className="h-3 w-3" />,
+          color: 'text-red-600 dark:text-red-400',
+          bgColor: 'bg-red-50 dark:bg-red-950/30',
+          badge:
+            'bg-red-100 dark:bg-red-900/50 text-red-800 dark:text-red-200 border-red-300 dark:border-red-700',
+        };
+      case 'info':
+      default:
+        return {
+          icon: <Info className="h-3 w-3" />,
+          color: 'text-gray-600 dark:text-gray-400',
+          bgColor: 'bg-gray-50 dark:bg-gray-950/30',
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)